upgrade_tests/thrift_upgrade_test.py

```diff
@@ -350,29 +469,41 @@ def test_sparse_supercolumn_with_renames(self):
         client.transport.open()
         client.set_keyspace('ks')
 
-        cf = _create_sparse_super_cf('sparse_super_1')
-        client.system_add_column_family(cf)
+        _create_sparse_super_cf(client, 'sparse_super_1')
+
+        #The alter after was failing claiming the column family didn't exist.
+        #This test is so slow and it's Thrift and going away "soon" and we
+        #Don't run it on every commit so just sleep
+        import time
+        time.sleep(5)
```

Comment:
Do we want to sleep for a fixed duration or keep retrying patiently until it succeeds?

Reply:
Removed entirely. I added the wait for schema agreement stuff so it shouldn't be necesssary